shopinvader_image/models/shopinvader_backend.py

```diff
@@ -1,9 +1,28 @@
 # Copyright 2017 Akretion (http://www.akretion.com).
 # @author Sébastien BEAU <sebastien.beau@akretion.com>
+# Copyright 2022 Camptocamp SA (http://www.camptocamp.com)
+# @author Simone Orsi <simahawk@gmail.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from odoo import fields, models
 
+CDN_HELP_TEXT = """
+Enable this flag to control
+whether the CDN URL must be included or not
+in the URL stored in images' JSON data.
+
+OFF: the URL will NOT include the CDN;
+ON: the URL will be include the CDN;
+
+Excluding the URL can be useful to:
+
+1. reduce payload size w/o duplicating dat
+2. allow using the same storage w/ different CDN
+
+If you use this option,
+take care of adding the CDN url on your frontend.
+"""
```

Comment:
Help message not clear enough.
Is it about relative path ?

Reply:
I improved the text.